compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -1238,7 +1237,8 @@ static void buildLLVMGPUCodegenConfigurationPassPipelineImpl(
   }
   modulePassManager.addPass(createMaterializeTuningSpecsPass());
   modulePassManager.addPass(createMaterializeUserConfigsPass());
-  modulePassManager.addPass(createLLVMGPUSelectLoweringStrategyPass());
+  modulePassManager.addPass(createLLVMGPUSelectLoweringStrategyPass(
+      {/*useDirectLoad=*/clUseDirectLoad}));
```

Comment:
Let's move the CLI flag to `KernelConfig.cpp`. That way we don't need to plumb `useDirectLoad` through everywhere. It's a bad sign that the API for CPU had to change for this. Ultimately we want to be making the decision for whether to use the direct load instructions based on the GPU target description, not CLI flags (and if we do have flags, they live in the target plugin, not here).